ethereum/networks/src/test/java/tech/pegasys/teku/networks/Eth2NetworkConfigurationTest.java

```diff
@@ -114,6 +147,15 @@ public void applyNamedNetworkDefaults_shouldOverwritePreviouslySetValues() {
             Eth2Network.LESS_SWIFT, (NetworkDefinition) b -> b.applyLessSwiftNetworkDefaults()));
   }
 
+  private List<NodeRecord> parseBootnodes(List<String> bootnodes) {
+    final NodeRecordFactory nodeRecordFactory = NodeRecordFactory.DEFAULT;
+
+    return bootnodes.stream()
+        .map(enr -> enr.startsWith("enr:") ? enr.substring("enr:".length()) : enr)
+        .map(nodeRecordFactory::fromBase64)
+        .collect(Collectors.toList());
```

Comment:
This is the right thing to do for now, but if you're up for another adventure, we could fix this in the discovery library (https://github.com/ConsenSys/discovery) so it automatically handles the `enr:` prefix in a new  `NodeRecordFctory.fromEnr` method instead of us having to do it all the time.

Reply:
Sure, I'm in! I created a new issue, https://github.com/ConsenSys/discovery/issues/99 . After the fix reaches the next version, I can do the additional simplifications.

Thank you for reviewing this PR, and for the suggestions!